product_sequence/data/product_sequence.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+  product_sequence for Odoo
+  Copyright 2004 Tiny SPRL
+  Copyright 2016 Sodexis
+  The licence is in the file __openerp__.py
```

Comment:
The license is in the file __manifest__.py

Reply:
You can remove this part if you want

Reply:
Or Add only `License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0)`